env.d.ts

```diff
@@ -7,10 +7,34 @@ declare module '*.vue' {
   export default component
 }
 declare global {
-  import type { SmartContractRequestParams } from '@/utils/signerMakeTx'
+  type SmartContractAcceptedValue = (string | undefined | number)[]
+
+  type TransactionSignerParams = {
+    appAddress: string
+    provider: any
+    forwarderAddress: string
+    gateway: string
+    accessToken: string
+  }
+
+  interface ImportMeta {
+    readonly env: any
```

Comment:
Can we expand this to type the env variables?

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/118/commits/f4c42b36050b13d9928f3ec362b10d5dfaadc9bd